components/session-service/server/server_functional_test.go

```diff
@@ -349,7 +358,7 @@ func TestMain(t *testing.T) {
 					Sub string `json:"sub"`
 					PU  string `json:"preferred_username"`
 				}{}
-				err = json.NewDecoder(resp.Body).Decode(&respMsg)
+				_ = json.NewDecoder(resp.Body).Decode(&respMsg)
```

Comment:
```suggestion
				json.NewDecoder(resp.Body).Decode(&respMsg)
```
Can we remove the left hand side completely ?

Reply:
Done